platform/app/public/config/default.js

```diff
@@ -13,6 +13,7 @@ window.config = {
   showLoadingIndicator: true,
   strictZSpacingForVolumeViewport: true,
   groupEnabledModesFirst: true,
+  disableConfirmationPrompts: false,
```

Comment:
It should be disabled by default. If someone wants it, they can enable it. However, the default setting should hide it.

Reply:
Sure, I thought we would keep it as it is, I’ll make confirmations hidden